editoast/src/views/infra/mod.rs

```diff
@@ -38,39 +38,36 @@ use editoast_derive::EditoastError;
 use futures::future::try_join_all;
 use futures::Future;
 use serde::{Deserialize, Serialize};
-use serde_json::{json, Value as JsonValue};
 use strum::IntoEnumIterator;
 use thiserror::Error;
+use utoipa::{IntoParams, ToSchema};
 use uuid::Uuid;
 
 /// Return `/infra` routes
 pub fn routes() -> impl HttpServiceFactory {
-    scope("/infra")
-        .service((list, create, refresh, cache_status, railjson::routes()))
-        .service(
-            scope("/{infra}")
-                .service((
-                    get,
-                    load,
-                    delete,
-                    clone,
-                    edit,
-                    rename,
-                    lock,
-                    unlock,
-                    get_switch_types,
-                    get_speed_limit_tags,
-                    get_voltages,
-                ))
-                .service((
-                    errors::routes(),
-                    objects::routes(),
-                    lines::routes(),
-                    routes::routes(),
-                    pathfinding::routes(),
-                    attached::routes(),
-                )),
-        )
+    services![
```

Comment:
Note to reviewer: the `scope("/{infra}")` is a problem here because if the URL parameters do not appear in the `#[get("")]` (or similar) annotations, utoipa doesn't detect the parameter. So we have to take the endpoints with a utoipa::path out of their scope. That part will change anyways when we'll implement a way to declare the paths and components to include in the openapi for just a module (instead of putting everything in the OpenApiRoot).